syntax = 'proto2';
package scalar;

message scalar{
  required int32 iter = 1;
  required double value = 2;
}

message full_scalar_it {
  repeated scalar scalar = 1;
}